src/kits/tracker/FindPanel.cpp

```diff
@@ -650,6 +768,76 @@ FindWindow::MessageReceived(BMessage* message)
 			Save();
 			break;
 
+		case kOpenSaveQueryPanel:
+		{
+			BSaveWindow *saveQueryWindow = new BSaveWindow(this);
+			saveQueryWindow->Show();
+			break;
+		}
+
+		case kCloseSaveQueryPanel:
+		{
+			const char *queryName;
```

Comment:
The * should be to the left, it is part of the type.

There are multiple occurrences in your code.

Reply:
Oh. I fixed this throughout the files. I will definitely keep this in mind.